### PR TITLE
first

### PR DESCRIPTION
**Pull request by Kocha1991**
- [DEMO LINK](https://Kocha1991.github.io/layout_catalog/)
- [TEST REPORT LINK](https://Kocha1991.github.io/layout_catalog/report/html_report/)

### CODE REVIEW

File: src/styles/elements/stars.scss [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  width: 16px;
+  height: 16px;
+  background-image: url("../../images/star.svg");
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars div:nth-child(-n + 4) {
+  background-image: url("../../images/star-active.svg");
+}
```
Comment (SanyaBratashchuk) [inline src/styles/elements/stars.scss:16]: reuse whole stars block from task with stars

File: src/styles/bloks/cards.scss [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+}
+
+.card__buy {
+  box-sizing: border-box;
+  display: block;
+  margin: 0 auto;
+  width: 166px;
```
Comment (ivanchuk-yurii) [inline src/styles/bloks/cards.scss:69]: don't hardcode width

File: src/styles/bloks/cards.scss [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__img {
```
Comment (ivanchuk-yurii) [inline src/styles/bloks/cards.scss:11]:
```suggestion
  &__img {
```
use nesting

File: src/styles/bloks/header.scss [hunk 4]
```diff
@@ -0,0 +1,20 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.is-active:after {
```
Comment (ivanchuk-yurii) [inline src/styles/bloks/header.scss:10]: you already have this in `active.scss`

File: src/styles/bloks/container.scss [hunk 5]
```diff
@@ -0,0 +1,10 @@
+.container {
+  margin: 0 auto;
+  padding-top: 3px;
+  max-width: 942px;
+  &__block {
```
Comment (ivanchuk-yurii) [inline src/styles/bloks/container.scss:5]:
```suggestion
  max-width: 942px;

  &__block {
```
calculate it based on card width

File: src/styles/elements/stars.scss [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background-image: url("../../images/star.svg");
+  background-repeat: no-repeat;
+  background-position: center;
```
Comment (ivanchuk-yurii) [inline src/styles/elements/stars.scss:11]: combine into `background` property

File: src/styles/elements/stars.scss (hunk 1 above)
Comment (ivanchuk-yurii) [reply src/styles/elements/stars.scss:16]: not fixed, it should work for any quantity of stars, don't use `div` selector

File: src/styles/variables/variables.scss [hunk 7]
```diff
@@ -0,0 +1,3 @@
+:root {
```
Comment (ivanchuk-yurii) [inline src/styles/variables/variables.scss:1]: use `scss` variables

File: src/styles/main.scss [hunk 8]
```diff
@@ -1,3 +1,26 @@
...
+@import url("./bloks/header.scss");
+@import url("./bloks/logo.scss");
+@import url("./bloks/active.scss");
+@import url("./bloks/cards.scss");
+
+@import url("./bloks/container.scss");
+
+@import url("./variables/variables.scss");
+
+:root {
```
Comment (ivanchuk-yurii) [inline src/styles/main.scss:11]: you already have this in `variables.scss`

File: src/styles/bloks/nav.scss [hunk 9]
```diff
@@ -0,0 +1,37 @@
...
+
+.nav__blue {
+  color: var(--blue);
+}
+
+.nav__link:hover {
+  color: var(--blue);
+}
+
+.nav__list li:first-child {
```
Comment (ivanchuk-yurii) [inline src/styles/bloks/nav.scss:35]: don't style tag here

File: src/styles/bloks/nav.scss [hunk 10]
```diff
@@ -0,0 +1,37 @@
+.nav__list {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  align-items: center;
+  list-style-type: none;
+}
+
+.nav__item {
+  position: relative;
```
Comment (ivanchuk-yurii) [inline src/styles/bloks/nav.scss:10]: it's better to move this to `.is-active`

**Pull request closed without merging**